update_snapcraft.py

```diff
@@ -62,15 +67,20 @@ def parse_manual_requirements(path):
     """
     manual_requirements = set()
 
-    if path.exists():
-        with open(path, encoding="utf-8") as req_file:
-            for line in req_file:
-                try:
-                    # Requirement cannot handle comments
-                    req = Requirement(line.split("#", maxsplit=1)[0])
-                    manual_requirements.add(str(req))
-                except InvalidRequirement:
-                    pass
+    # This provides a second default layer: since the GitHub workflow provides
+    # a fixed path in the snap-tempest checkout, that path may not actually
+    # exist. When that happens, we can use the list provided in this repo as a
+    # non-per-release default
+    manual_requirements_path = path if path.exists() else MANUAL_REQUIREMENTS_PATH
+
+    with open(manual_requirements_path, encoding="utf-8") as req_file:
+        for line in req_file:
+            try:
+                # Requirement cannot handle comments
+                req = Requirement(line.split("#", maxsplit=1)[0])
+                manual_requirements.add(str(req))
+            except InvalidRequirement:
+                pass
```

Comment:
Not really related to your change, but it will be nice to log / print when there are some `InvalidRequirement` for troubleshooting

Reply:
where / how do you think we should log? This script is meant to be used in a workflow, so a print would easily be overlooked

Reply:
on a second thought, maybe we should crash the tool? since if it's not a valid requirement or missing requirement(because it's not typed correctly), the snap might not work properly.

Reply:
yeah that's fine. TBH by dropping the monasca plugin this functionality is no longer needed, but since it's already hear I thought of leaving it in case we need it again in the future

Reply:
okay then, just a print / log to the workflow should be enough as long as we can find it when we want to (or ignore it this comment, this feature might be removed later). Feel free to resolve this comment